.github/workflows/monitor_self_hosted_runners.yml

```
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Monitor Self-Hosted Runners Status
+on:
+  schedule:
+    - cron: "0 */12 * * *"
```

```suggestion
    - cron: "*/30 * * * *"
```

Lets maybe run this more frequently since it should be cheap?

---

Yes, it's only a single Cloud Function call so we can run it more frequently